### DISCUSSION TITLE
using different image size while using a pre trained model

### DISCUSSION THREAD

**Discussion by Md-Sayeed-Khan** [mrdbourke/pytorch-deep-learning#849] [Q&A]
When using a pretrained model, it's important that your custom data going into the model is prepared in the same way as the original training data that went into the model.

why is it important ?
 i tried to use different image size while training and it worked good . 
also while predicting on un seen images i did not resize to size which model was trained on , i passed directly an high resolution image and it worked fine

**Comment by LuluW8071** [comment]
By pre trained model from your point of view, I guess you are asking about transfer learning through pre saved model weights. 

If so, Resizing images isn't strictly necessary but is often done primarily to reduce image feature complexity and training time.  Resizing ensures consistency with the dataset used to train the pretrained model.
